scriptworker/cot/verify.py

```diff
@@ -1217,13 +1217,19 @@ def verify_link_in_task_graph(chain, decision_link, task_link):
     ).get_commit(commit_hash)
 
     committer_login = commit_data['committer']['login']
-    committer_email = commit_data['commit']['committer']['email']
+    # committer_email = commit_data['commit']['committer']['email']
```

Comment:
Nit: let's remove these commented out lines in order to not confused future readers.